bower.json

```diff
@@ -21,6 +21,6 @@
         "spec"
     ],
     "dependencies": {
-        "moment": "~2.10.3"
+        "moment": "~2.12"
```

Comment:
I have fairly little experience with Bower and would appreciate assistance in confirming that any updates work correctly.  (Some simple script I could run would be great!)